### ISSUE TITLE
Consider adding the title of a tool to the when the bot creates github issues

### ISSUE THREAD

**Issue by Greenheart** [Greenheart/idg.tools#220]
By adding the title of the tool in the GitHub issue title, it would be easier as a maintainer to quickly take action.

We could also create a basic guide for how to triage and prioritize among the incoming issues.

This should be a simple technical fix, adding the tool's title to be added to the end of this line: https://github.com/Greenheart/idg.tools/blob/main/tools/src/lib/github.ts#L68

**Comment by letzfets** [comment]
Yes, I see the value in passing the title to make the issues more readable and easier accessible.

### passing the title

Technically, I'm not sure, if the title of the tool is available in [https://github.com/Greenheart/idg.tools/blob/main/tools/src/lib/github.ts#L68](https://github.com/Greenheart/idg.tools/blob/main/tools/src/lib/github.ts#L68)

Is it necessary to pass `tool.name` into `FeedbackForm` here [https://github.com/Greenheart/idg.tools/blob/038fde15b5adc01866c365504a8303734a5b1b2c/tools/src/routes/%5Blink%5D/%2Bpage.svelte#L54](https://github.com/Greenheart/idg.tools/blob/038fde15b5adc01866c365504a8303734a5b1b2c/tools/src/routes/%5Blink%5D/%2Bpage.svelte#L54)

and further add the title to the form data in the `enhance` function 

[https://github.com/Greenheart/idg.tools/blob/038fde15b5adc01866c365504a8303734a5b1b2c/tools/src/components/FeedbackForm.svelte#L21](https://github.com/Greenheart/idg.tools/blob/038fde15b5adc01866c365504a8303734a5b1b2c/tools/src/components/FeedbackForm.svelte#L21) ?

### testing

I agree: testing all the way is trouble some, but in a development environment without the `GITHUB_API_TOKEN` the request would fail. And it should be good enough to log the message

### formating the date

I'd also suggest to make the time format shorter: date, hours and minutes seem adequate to me - millisecond precision might be too much information. Maybe date and time would be more valuable together with the URL and git commit tag in the body of the message here. [https://github.com/Greenheart/idg.tools/blob/038fde15b5adc01866c365504a8303734a5b1b2c/tools/src/lib/github.ts#L58](https://github.com/Greenheart/idg.tools/blob/038fde15b5adc01866c365504a8303734a5b1b2c/tools/src/lib/github.ts#L58)

We could also consider moving the technicalities, like URL, git commit tag and date and time to the bottom of the message and start the message with the essential input from the user? 🤔

**Comment by Greenheart** [comment]
Great input! I agree with all suggestions about moving the data fields to the bottom of the post 😊

Also, we kind of already have the time since that's when the issue gets created. So we could save data by removing it entirely. Let's remove the date and time completely.

**Comment by Greenheart** [comment]
Regarding passing data like the title down to the feedback form, we could pass the full tool like this

<FeedbackForm {tool} /> which is equlvalent to <FeedbackForm tool={tool} /> that might be a more familiar syntax from React for example.
